subworkflows/local/standardisation_profiles.nf

```diff
@@ -2,8 +2,10 @@
 // Standardise output files e.g. aggregation
 //
 
-include { KAIJU_KAIJU2TABLE                     } from '../../modules/nf-core/modules/kaiju/kaiju2table/main'
-include { MOTUS_MERGE } from '../../modules/nf-core/modules/motus/merge/main'
+include { KRAKENTOOLS_COMBINEKREPORTS as KRAKENTOOLS_COMBINEKREPORTS_CENTRIFUGE } from '../../modules/nf-core/modules/krakentools/combinekreports/main'
+include { KAIJU_KAIJU2TABLE                                                     } from '../../modules/nf-core/modules/kaiju/kaiju2table/main'
+include { KRAKENTOOLS_COMBINEKREPORTS                                           } from '../../modules/nf-core/modules/krakentools/combinekreports/main'
```

Comment:
How about to define this as well for kraken? Or is it unnecessary?

Reply:
I decided to leave it as is, because the centrifuge is a bit of a 'hack ' . `kreport` implies kraken to me. But if you think it's better to have both for consistency I'm happy to add it!

Reply:
I was thinking more for consistency but I think that it is self-explanatory as it is now as well.